proposals/2021-09-hub-ibc-router/README.md

```diff
@@ -0,0 +1,25 @@
+# ⚛️ Make the Cosmos Hub the IBC Router ⚛️
+
```

Comment:
```suggestion

This is a proposal to include a new feature to IBC that allows a simplified version of multi-hop packet routing. This version is simplified in that it only works for token transfers. By appending a list of channels, ports and addresses the destination can outline more than one transfer at a time. Each set of channel, port and destination addresses is utilized until the sequence is exhausted. If one of the hops fails, the tokens stay within the address used in the last successful transfer, or goes back to the sender address should it fail on the first hop.

```

Reply:
I thought a higher level summary of the work might be good as @jtremback suggested as well. Do you think this attempt doesn't capture the intention well?